benchmark/util/text-encoder.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const common = require('../common.js');
+
+const BASE = 'string\ud801';
+
+const bench = common.createBenchmark(main, {
+  n: [1e4],
+  op: ['encode', 'encodeInto']
+});
+
+function main({ n, ignoreBOM, op }) {
```

Comment:
```suggestion
function main({ n, op }) {
```

Reply:
Thanks! I just updated the pull request.